style.css

```diff
@@ -890,6 +1007,121 @@ a:active {
     align-items: center;
     width: 20%;
   }
+
+  /* Modal section starts here */
+
+  .modal__container {
+    width: 100vw;
+    height: 100%;
+    background: #8193af;
+    position: fixed;
+    top: 0;
+    display: none;
+    backdrop-filter: blur(70px);
+  }
+
+  .modal__box {
+    width: 70%;
+    margin: 0 auto;
+    align-items: center;
+    padding-top: 1%;
+    background: white;
+    display: flex;
+    flex-direction: column;
+    height: 95vh;
+    overflow-y: scroll;
+  }
```

Comment:
- [ ] Hi @Besufekad-HAZ, great effort working on the modal popup of your portfolio project, 👍. But I would like to raise some of the issues you have to pay attention to in this milestone. As it is required that we follow the project requirement and the Figma design as much as possible, below are the highlight of what is needed to work on.

   - The popup card is hitting the top of the page kindly give some margin to the top ``e.g margin: 20px auto;``
   - I would kindly suggest you remove the scroll bar around the `card title`, `see live`, and `see source` buttons to look exactly like what we have in the Figma design.
   - Kindly give padding around the card content, don't limit the padding to the top alone ``e.g padding:1.5% 1%;``
   - I would suggest you remove the `overflow-y: scroll;` on line 1033 and your popup will look just fine like what we have in the Figma Design.
   - I would kindly suggest you give your card some `backdrop-filter: blur(10px);` CSS property so that the main page looks blurred when the popup card is displayed. I notice you gave the class `.modal_container) the backdrop filter property but it's not working, then I would advise you to make use of the RBGA color property for the background. Try this and see if it would work.
   
   
Your Design 👇 

![besu](https://user-images.githubusercontent.com/78634841/216429905-c75f4885-bff4-4420-abd0-4c08bcf20dc7.png)

Figma Design 👇 

![pp3-real](https://user-images.githubusercontent.com/78634841/216430037-c41d583e-ab91-4a6e-8699-4f1a94cae925.png)

Reply:
Ok...Thanks for the constructive Feedback!!
I'll definitely fix these issues.